src/server/api/index.js

```diff
@@ -142,8 +143,9 @@ router.get('/analyticsfile', (req, res) => {
     }
     Analytics.getAnalytics().then(result => {
       const excel = json2xls(result);
-      fs.writeFileSync('analytics.xlsx', excel, 'binary');
-      res.download('analytics.xlsx');
+      const tmpobj = tmp.fileSync();
+      fs.writeFileSync(tmpobj.name, excel, 'binary');

```

Comment:
@finnzink This will still write the file in the current directory right? It should write it to a separate folder

Reply:
@rlorenzo Ok so I was looking into this and it turns out that tmp writes the file into a newly created directory /var/folders/ which it cleans up every time. I think I would be fighting the library to try and write to a custom folder. The fileSync method doesn't seem to support custom directories. Is that sufficient?